nwbib/nwbib.ttl

```diff
@@ -7715,68 +8543,81 @@
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:narrower :N882020, :N882030, :N882040, :N882060, :N882090 ;
     skos:notation "882000" ;
-    skos:prefLabel "Publizistik"@de .
+    skos:prefLabel "Publizistik"@de ;
+    skos:exactMatch gnd:4047770-8 .
 
 :N882020
     a skos:Concept ;
     skos:broader :N882000 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:narrower :N882022, :N882024, :N882026 ;
     skos:notation "882020" ;
-    skos:prefLabel "Presse"@de .
+    skos:prefLabel "Presse"@de ;
+    skos:exactMatch gnd:4047150-0, wd:Q1193236 .
 
 :N882022
     a skos:Concept ;
     skos:broader :N882020 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "882022" ;
-    skos:prefLabel "Presserecht"@de .
+    skos:prefLabel "Presserecht"@de ;
+    skos:exactMatch gnd:4047171-8, wd:Q7241640 .
 
 :N882024
     a skos:Concept ;
     skos:broader :N882020 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "882024" ;
-    skos:prefLabel "Zeitungen"@de .
+    skos:prefLabel "Zeitungen"@de ;
+    skos:exactMatch gnd:4067510-5, wd:Q11032 .
 
 :N882026
     a skos:Concept ;
     skos:broader :N882020 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "882026" ;
-    skos:prefLabel "Zeitschriften"@de .
+    skos:prefLabel "Zeitschriften"@de ;
+    skos:exactMatch gnd:4067488-5, wd:Q41298 .
 
 :N882030
     a skos:Concept ;
     skos:broader :N882000 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "882030" ;
-    skos:prefLabel "Audiovisuelle Medien"@de .
+    skos:prefLabel "Audiovisuelle Medien"@de ;
+    skos:exactMatch gnd:4068939-6, wd:Q758904 .
 
 :N882040
     a skos:Concept ;
     skos:broader :N882000 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "882040" ;
-    skos:prefLabel "Rundfunk"@de .
+    skos:prefLabel "Rundfunk"@de ;
+    skos:exactMatch gnd:4050963-1, wd:Q15078788 .
 
 :N882060
     a skos:Concept ;
     skos:broader :N882000 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "882060" ;
-    skos:prefLabel "Fernsehen"@de .
+    skos:prefLabel "Fernsehen"@de ;
+    skos:altLabel "Fernsehsendung, TV, Fernseh-Rundfunk, Personen aus dem Bereich Fernsehen"@de ;
+    skos:exactMatch gnd:4016842-6, wd:Q289 ;
+    skos:closeMatch gnd:4016825-6 .
 
 :N882090
     a skos:Concept ;
     skos:broader :N882000 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "882090" ;
-    skos:prefLabel "Journalisten"@de .
+    skos:prefLabel "Journalisten"@de ;
+    skos:exactMatch gnd:4028781-6, gnd:4028786-5, wd:Q1930187 .
```

Comment:
`skos:narrowMatch` für GND-Links?

Reply:
Bei 882090 Journalisten wollte ich sowohl die weibliche als auch die männliche Form benennen. Dafür kann ich mir gut vorstellen skos:altLabel zu verwenden. Das kann dann in der Suche innerhalb der Sachsystematik eingebunden werden, sodass man bei der Eingabe von Journalistin auch auf die Systemstelle 882090 Journalisten kommt. Bei der Videokonferenz mit Münster und Bonn hatten wir ja beschlossen, dass die Suche im Suchschlitz der Sachsystematik auf diese Weise ausgebaut werden kann (auch mit aktualisierten Bezeichnungen: Suche nach Tourismus -- Ergebnis: Fremdenverkehr).

Reply:
s.a. Verleger, Bibliothekare, Künstler...

Reply:
Der STW löst es so: https://zbw.eu/stw/version/latest/descriptor/13995-4/about.de.html
Ah und da sind die weiblichen Formen unter narrowmatch https://zbw.eu/stw/version/latest/descriptor/13995-4/about.rdf

Bei Künstler wiederum sind die männliche und weibliche Form exactMatch : https://zbw.eu/stw/version/latest/descriptor/13355-5/about.de.html